src/react-compat.js

```diff
@@ -150,6 +157,16 @@ if (REACT013) {
   };
 }
 
+if (REACT155) {
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  PropTypes = require('prop-types');
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  createClass = require('create-react-class');
```

Comment:
@acdlite fwiw, if both of these packages happened to work down to react 0.13, we could drop a lot of our crappy conditional requires here :-)